<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conda</class>
 <widget class="QMainWindow" name="Conda">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codna</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_conda_tip">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Conda可执行文件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_conda">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>256</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>25</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_envs">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_current">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>使用现有环境</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_new">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>创造新环境</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_env_name_tip">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>环境名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_env_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>25</height>
     </size>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_version_tip">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Python版本：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_version">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="currentText">
     <string>3.11</string>
    </property>
    <item>
     <property name="text">
      <string>3.11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.6</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_select</sender>
   <signal>clicked()</signal>
   <receiver>Conda</receiver>
   <slot>select_conda()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_envs</sender>
   <signal>activated(int)</signal>
   <receiver>Conda</receiver>
   <slot>update_env_name()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_conda()</slot>
  <slot>update_env_name()</slot>
 </slots>
</ui>
